kano_updater/scenarios.py

```diff
@@ -332,3 +332,12 @@ def beta_201_to_beta_210(self):
 
     def beta_210_to_beta_220(self):
         install('telnet python-serial')
+        try:
+            from kano_profile.apps import save_app_state_variable
+
+            save_app_state_variable("init-flow-completed", "level", 1)
```

Comment:
This will only happen for the user who is running the updater, not for all of the installed users in the system